src/main/java/com/revature/util/ExceptionAspect.java

```diff
@@ -11,18 +11,15 @@
 @Component
 @Aspect
 public class ExceptionAspect { //Aspect handling exceptions
-@Autowired
-Logging log;
+	@Autowired
+	public Logging log;
```

Comment:
What was the reason for this change - Does this need to be public?

Reply:
No, it does not need to be public. This was a temporary change that Andres made. As I mentioned before, the project was not following best practices in terms of test package naming. So He was not able to access the Logging object. I told him to rename the test package so he can access the class, so the field can be default package-private. He simply forgot to change the access modifier.

Reply:
Got it, can we remove this if it's not necessary ?

Reply:
And update the test package names?